resources/terminology/ContactPointSystem-zib-ContactInformation-TelephoneNumbers.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="ContactPointSystem-zib-ContactInformation-TelephoneNumbers"/>
+    <url value="http://nictiz.nl/fhir/ValueSet/ContactPointSystem-zib-ContactInformation-TelephoneNumbers"/>
+    <version value="4.0.1"/>
+    <name value="contactPointSystemZibContactInformationTelephoneNumbers"/>
```

Comment:
Volgens mij gaat het niet niet helemaal goed met de guidelines. In de ConceptMaps wordt vervolgens deze naam gebruikt `ContactPointUse_zib_ContactInformation_TelephoneNumbers`

Ook wel echt een hele moeilijke naam: kunnen we dit wat inkorten, bijv ContactPointSystem-Telephone?

Reply:
Het is namelijk nogal een dingetje:

![image](https://user-images.githubusercontent.com/20518398/109986763-57212d80-7d06-11eb-9b87-be829d5ddbdf.png)

Reply:
Over ValueSets die niet van A-D komen hebben we niks afgesproken. Dit is natuurlijk niks, "ContactPoinstSystem-Telephone" is weer nikszeggend. Mss "zib-ContactInformation-TelephoneSystem", "zib-ContactInformation-TelephoneUse", "zib-ContactInformation-E-mailUse"?

Reply:
Ik voel wel wat voor de suggesties die jij geeft! Laten we daarvoor gaan.

Reply:
@ArdonToonstra  ik heb het nu flink ingekort, akkoord?

Reply:
Ook nog "zib" eraf gehaald omdat dat wellicht verwarring kan wekken tov. de zib-profielen.

Reply:
@pieter-edelman-nictiz , ja! Echt veel beter.